.travis.yml

```diff
@@ -29,6 +29,7 @@ matrix:
       env: DEPENDENCIES='dev'
 
 before_install:
+  - echo "memory_limit=4G" >> ~/.phpenv/versions/$(phpenv version-name)/etc/conf.d/travis.ini
```

Comment:
looks like we can *now* use the `COMPOSER_MEMORY_LIMIT` var for this (see https://github.com/composer/composer/issues/6931#issuecomment-355240659)

Reply:
That's new to me, I'll have a look.

Reply:
Looks like we sadly cannot use it now as the current release has some autoloading issues (see #1833). So let's stick with your approach for the moment.